### ISSUE TITLE
keybase login doesn't work

### ISSUE THREAD

**Issue by lheckemann** [NixOS/nixpkgs#21925]
## Issue description
```
$ keybase login
<type details>
ERROR no auth system available; consider upgrading your client if it's old (error 100)
```

## Technical details

* System: 16.09.1465.fd1dbe5 (Flounder)
* Nix version: nix-env (Nix) 1.11.4
* Nixpkgs version: 16.09.1465.fd1dbe5

This is fixed in the latest version of keybase; cherry-picking 711a42e03aa44439142bb820319c0d61a6faa7ae onto my copy of nixpkgs and updating keybase fixed the issue for me.

**Comment by globin** [comment]
Fixed in 5de9698e7b

**Comment by lheckemann** [comment]
Wow, that was rapid. Thanks!
